docs/examples/pytorch/single_stage_detector/src/coco_pipeline.py

```diff
@@ -57,8 +57,12 @@ def __init__(self, default_boxes, args, seed):
             allow_no_crop=True,
             num_attempts=1)
         self.slice = ops.Slice(device="cpu")
-        self.bc = ops.BrightnessContrast(device="gpu")
-        self.hsv = ops.Hsv(device="gpu")
+
+        self.hsv = ops.Hsv(device="gpu", output_dtype=types.FLOAT)
```

Comment:
I would add some comment that we need data between ops.Hsv and ops.BrightnessContrast in float to preserve accuracy. The latter commend doesn't address that directly.